### PR TITLE
Update the Remote webpage

### PR DESCRIPTION
**Pull request by lumaa-dev**
This update comes with the following changes:
1. An updated screenshot of the Remote app
2. A "Latest Features" section for the future new features
    - Currently, the last introduced feature is the "Live Activity"
3. A cropped screenshot of the "Live Activity" latest feature
- Also updated the minimum required iOS version to the one associated with it in the [Xcode project](https://github.com/ciderapp/Cider-Remote/blob/main/Cider%20Remote.xcodeproj/project.pbxproj#L761); Here being iOS 16.1

### CODE REVIEW

**Comment by Monochromish** [comment]
Hey Luna, Thank you for the changes you made; they look terrific. I just have one suggestion:

To be more consistent with the design of other pages, could you remove the second picture and switch the positions of the "Many more features" card with the "Live activities" card? That should make it look better and remove additional whitespace.

You can also remove the "Latest Features" button for redundancy, as we already have a Changelog button.

File: public/remote_player.png
Comment (Monochromish) [inline public/remote_player.png:1]: Could use iPhone 16 screenshot

Comment (lumaa-dev) [reply public/remote_player.png:1]: I don't own an iPhone 16

Comment (Monochromish) [reply public/remote_player.png:1]: Do you need to own one to try Cider Remote on simulator?

**Review by Monochromish — APPROVED** [review]

**Comment by lumaa-dev** [comment]
After making the necessary changes, here is the result:

**Comment by Monochromish** [comment]
Thank you once again, Luna. You're awesome!

**Comment by coredev-uk** [comment]
For future reference, please maintain the commit message standard we outline in our contributing guide.

**Pull request merged**